### PR TITLE
feat: migrate to Angular 8

### PR DESCRIPTION
**Pull request by VladimirAmiorkov**
Testing the current RC of Angular with `nativeScript-angular`

Fixes: https://github.com/NativeScript/nativescript-angular/issues/1833

BREAKING CHANGES:

 - In Angular version 8, it's required that all @ViewChild and @ContentChild queries have a 'static' flag specifying whether the query is 'static' or 'dynamic'. More details about this change can be found [here](https://angular.io/guide/static-query-migration).

Migration steps:

Anywhere you previously had `@ViewChild` with a single param you now have to provide a second param with a `static` property set to either `true` or `false`

**Previous code:**
```
import { ElementRef } from "@angular/core";

@ViewChild("myElement") myElement: ElementRef;
```

**Migrated code:**
```
import { ElementRef } from "@angular/core";

@ViewChild("myElement", { static: false }) myElement: ElementRef;
```

### CODE REVIEW

**Comment by SvetoslavTsenov** [comment]
test nativescript_dev_webpack#/angular-8-rc sdk_ng sample_Groceries

**Comment by SvetoslavTsenov** [comment]
test nativescript_dev_webpack#angular-8-rc sdk_ng sample_Groceries

**Comment by DimitarTachev** [comment]
test nativescript_dev_webpack#feature/angular8 sdk_ng sample_Groceries

**Comment by SvetoslavTsenov** [comment]
test build_nativescript_dev_webpack#feature/angular8 sdk_ng sample_Groceries

**Comment by SvetoslavTsenov** [comment]
test build_nativescript_dev_webpack#feature/angular8 sdk_ng sample_Groceries

**Comment by SvetoslavTsenov** [comment]
test branch_nativescript_dev_webpack#feature/angular8 sdk_ng sample_Groceries

**Comment by SvetoslavTsenov** [comment]
test branch_nativescript_dev_webpack#feature/angular8 sdk_ng sample_Groceries

**Comment by SvetoslavTsenov** [comment]
test --ignore api23 ios11 branch_nativescript_dev_webpack#feature/angular8 sdk_ng#amiorkov/angular8-rc sample_Groceries#amiorkov/angular8-rc

**Comment by SvetoslavTsenov** [comment]
test --ignore api23 ios11 branch_nativescript_dev_webpack#feature/angular8 sdk_ng#amiorkov/angular8-rc groceries#amiorkov/angular8-rc

**Comment by VladimirAmiorkov** [comment]
test --ignore api23 ios11 branch_nativescript_dev_webpack#feature/angular8 sdk_ng#amiorkov/angular8-rc groceries#amiorkov/angular8-rc

**Comment by VladimirAmiorkov** [comment]
test --ignore api23 ios11 branch_nativescript_dev_webpack#feature/angular8 sdk_ng#amiorkov/angular8-rc groceries#amiorkov/angular8-rc

**Comment by VladimirAmiorkov** [comment]
test --ignore api23 ios11 branch_nativescript_dev_webpack#feature/angular8 sdk_ng#amiorkov/angular8-rc groceries#amiorkov/angular8-rc

**Comment by VladimirAmiorkov** [comment]
test --ignore api23 ios11 branch_nativescript_dev_webpack#feature/angular8 sdk_ng#amiorkov/angular8-rc groceries#amiorkov/angular8-rc

**Comment by VladimirAmiorkov** [comment]
test --ignore api23 ios11 branch_nativescript_dev_webpack#feature/angular8 sdk_ng#amiorkov/angular8-rc groceries#amiorkov/angular8-rc

**Comment by VladimirAmiorkov** [comment]
test --ignore

File: e2e/nested-router-tab-view/package.json [hunk 1]
```diff
@@ -6,22 +6,22 @@
   "nativescript": {
     "id": "org.nativescript.nestedroutertabview",
     "tns-ios": {
-      "version": "next"
+      "version": "6.0.0-2019-05-16-165318-01"
```
Comment (vakrilov) [inline e2e/nested-router-tab-view/package.json]: leave this as `next`

Comment (VladimirAmiorkov) [reply e2e/nested-router-tab-view/package.json]: Yes, this was incorrect commit caused by CLI auto changing this tag. I have reverted it.

**Comment by VladimirAmiorkov** [comment]
test --ignore

**Comment by VladimirAmiorkov** [comment]
test --ignore

**Pull request merged**